uaclient/security.py

```diff
@@ -1138,15 +1174,13 @@ def _prompt_for_attach(cfg: UAConfig) -> bool:
     _inform_ubuntu_pro_existence_if_applicable()
     print(messages.SECURITY_UPDATE_NOT_INSTALLED_SUBSCRIPTION)
     choice = util.prompt_choices(
-        "Choose: [S]ubscribe at ubuntu.com [A]ttach existing token [C]ancel",
+        messages.SECURITY_FIX_ATTACH_PROMPT,
         valid_choices=["s", "a", "c"],
     )
     if choice == "c":
         return False
     if choice == "s":
-        print(messages.PROMPT_UA_SUBSCRIPTION_URL)
-        # TODO(GH: #1413: magic subscription attach)
-        input("Hit [Enter] when subscription is complete.")
+        return _perform_magic_attach(cfg)
     if choice in ("a", "s"):
```

Comment:
I guess "s" can be removed here since now it returns early - we can make it explicit.

Reply:
@renanrodrigo done